### ISSUE TITLE
[Accessibility][Navigation][Language Navigation] Link text announced twice

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1670]
**Steps to Reproduce**
1. With NVDA running, open the page in Chrome browser.
2. Using the Tab key, navigate through list items in the Language Navigation component.

**Current Behavior**
With NVDA running, the name of each navigation item is announced twice (e.g. "Config  link    Config"). An unnecessary `title` attribute duplicates the visible link text.

Note: In the Language Navigation component, for links in a language other than the page default, this results in the text of the link being pronounced correctly (i.e. with the right speech synthesizer), followed by the link being pronounced incorrectly (with the default speech synthesizer), because NVDA does not use the `lang` attribute of an element to select the speech synthesizer for the `title` attribute.

**Expected behavior/code**
The text of each link should be announced only once to avoid repetition for screen reader users. 

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
2.2.4 Link Text (In Context)

**Possible Solution**
Consider removing the `title` attribute on each link.

**Code snippet**
```
<!-- from Navigation component -->
<a href="/content/core-components-examples/config.html" title="Config" data-cmp-clickable="" class="cmp-navigation__item-link">Config</a>

<!-- from Language Navigation Component -->
<a data-cmp-clickable="" class="cmp-languagenavigation__item-link" href="/content/core-components-examples/library/templating/language-navigation/language-structure/fr/fr/language-navigation.html" hreflang="fr-FR" lang="fr-FR" rel="alternate" title="Français">Français</a>
```
**Additional context**
NVDA Bug: NVDA does not use the lang attribute when announcing the title attribute of an element -- https://github.com/nvaccess/nvda/issues/8874

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by bpauli** [comment]
done in https://github.com/adobe/aem-core-wcm-components/pull/1698
